Dockerfile.base

```
@@ -41,6 +41,7 @@ RUN echo en_US.UTF-8 UTF-8 >> /etc/locale.gen \
     python3-setuptools \
     python3 \
     dumb-init \
+  && sed -e 's/Defaults env_reset/Defaults  env_keep = "HTTP_PROXY HTTPS_PROXY NO_PROXY FTP_PROXY http_proxy https_proxy no_proxy ftp_proxy"/' -i /etc/sudoers \
```

`Defaults env_reset` is fragile. it failed for me locally because of extra spaces:

```
root@1d521066b4d4:/# cat /etc/sudoers | grep env_reset
Defaults        env_reset
```

I'd use `.*` instead of a single space

```suggestion
  && sed -e 's/Defaults.*env_reset/Defaults  env_keep = "HTTP_PROXY HTTPS_PROXY NO_PROXY FTP_PROXY http_proxy https_proxy no_proxy ftp_proxy"/' -i /etc/sudoers \
```

---

Good catch. My tabs became spaces between copy and pasting between my local and this PR. I've updated it now.